README.md

```diff
@@ -10,15 +10,24 @@ Pre-requisites: `oc`, `jq` and `git` should be pre-installed and you should be l
 
 | :warning: WARNING                                                                                     |
 |-------------------------------------------------------------------------------------------------------|
-| Run `oc login` as an administrator on the target OpenShift cluster before running the following steps.|
-
-
+| Run `oc login` as an administrator on the target OpenShift cluster before running the following steps, or use the `--kubepwd` and `--openshift` flags to log in as kubeadmin automatically.|
```

Comment:
We don't want to provide a command line (we assume user knows how to do that). We warn that the following commands work only if the user 1) is connected to an openshift cluster 2) it has admin privileges.